### PR TITLE
Show simple summary

### LINKED ISSUE #26: button for summary of timereport
An idea is to have a button on the website (UI) to download a report for a given month in some format so it can be sent to for example our economy department for when they do the payroll each month.

Today I have to go in -> copy paste it in to a E-mail and send it.

Any other suggestions are welcomed.

### PR DESCRIPTION
**Pull request by mattiaslundberg**
Add a page that contains an easily copy-pasteable format for exporing to others. Also includes button to easily copy :)

Fix #26

### CODE REVIEW

File: src/components/Summary.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  };
+
+  return (
+    <div className="summary-container">
+      <DatePicker handleDateChange={handleDateChange} />
+
+      <button className="summary-copy" onClick={onCopy}>
+        Copy
+      </button>
+      {copied ? <span style={{ color: "red" }}>Copied.</span> : null}
```
Comment (parberge) [inline src/components/Summary.js:85]: This never gets cleared. I.e it will display the same "copied" message when you select a new month.

**Review by parberge — COMMENTED** [review]
Small fix about user experience suggested.

Also could we instead of showing the "copy" button when there is no data to copy just write a message about it. Like "No data to copy for month X".

File: src/components/Summary.js (hunk 1 above)
Comment (mattiaslundberg) [reply src/components/Summary.js:88]: Added a timer to hide it after 5 seconds

File: src/components/Summary.js [hunk 2]
```diff
@@ -71,7 +71,10 @@ export function Summary({ backendUrl }) {
   const onCopy = (e) => {
     navigator.clipboard
       .writeText(summaryRef.current.innerText)
-      .then(() => setCopied(() => true))
+      .then(() => {
+        setCopied(() => true);
+        setTimeout(() => setCopied(() => false), 5000);
```
Comment (parberge) [inline src/components/Summary.js:76]: If I understand this correctly the message will be displayed for 5 seconds?
I think it's too long. 1 sec should be enough.
```suggestion
        setTimeout(() => setCopied(() => false), 1000);
```

Comment (mattiaslundberg) [reply src/components/Summary.js]: 👍

**Review by parberge — APPROVED** [review]
👍 Maybe not the prettiest, but since I couldn't do better I still think this is awesome 😸

**Pull request merged**